api/mapping/views.py

```diff
@@ -1845,7 +1845,9 @@ def list(self, request, pk):
         # scan_report = ScanReportSerializer(scan_reports, many=False).data
         # Set Storage Account connection string
         print(scan_report)
+        # TODO: `name` is not always defined, it seems
```

Comment:
As I recall, we used to use the `file` field (`FileField`), but now we use `name` (`CharField`) to hold the filename. One more thing to add to the "must clean up" pile after 2.0 ;)